src/Forms/Prism.Forms/Navigation/PageNavigationService.cs

```diff
@@ -852,9 +852,9 @@ protected virtual Page CreatePageFromSegment(string segment)
                 throw new NavigationException(NavigationException.NoPageIsRegistered, _page, innerException);
             }
 
-            PageUtilities.SetAutowireViewModel(page);
-            _pageBehaviorFactory.ApplyPageBehaviors(page);
-            ConfigurePages(page, segment);
+                PageUtilities.SetAutowireViewModel(page);
+                _pageBehaviorFactory.ApplyPageBehaviors(page);
+                await ConfigurePages(page, segment);
```

Comment:
not sure what's up with the spacing here, but this should be reset to be at the right indent level

Reply:
fixed